zfile_windows.go

```diff
@@ -0,0 +1,27 @@
+// mksyscall_windows -l32 file_windows.go
+// MACHINE GENERATED BY THE COMMAND ABOVE; DO NOT EDIT
```

Comment:
Should we have a file with the corresponding go:generate call?

Reply:
That's a possibility... but at the same time, this code should never change, so maybe not?

Reply:
Okay. I wasn’t sure under which circumstances this code might change. If the compatibility guarantee is that it keeps working indefinitely, even on new windows versions, I’m fine without the go:generate statement :)